tests/e2e/cluster_downgrade_test.go

```diff
@@ -165,7 +177,7 @@ func testDowngradeUpgrade(t *testing.T, numberOfMembersToDowngrade int, clusterS
 	err = e2e.DowngradeUpgradeMembersByID(t, nil, epc, membersToChange, currentVersion, lastClusterVersion)
 	require.NoError(t, err)
 	if len(membersToChange) == len(epc.Procs) {
-		e2e.AssertProcessLogs(t, leader(t, epc), "the cluster has been downgraded")
+		e2e.AssertProcessLogs(t, epc.Procs[epc.WaitLeader(t)], "the cluster has been downgraded")
```

Comment:
Can we also call `e2e.ValidateDowngradeInfo` to check the downgrade info, to ensure that the downgrade can be automatically cancelled?

Reply:
release 3.5 doesn't have this change yet. so we can't do that

Reply:
If this is resolved, then we might not need https://github.com/etcd-io/etcd/pull/19399 any more

Reply:
> then we might not need #19399 any more

It seems that https://github.com/etcd-io/etcd/pull/19399 doesn't make any sense?

Reply:
I think so.

Reply:
FYI. @henrybear327 then let's close https://github.com/etcd-io/etcd/pull/19399